docs/UserGuide.md

```diff
@@ -213,20 +214,20 @@ Examples:
 * `list -i` followed by `delete -i 2` deletes the 2nd internship in InterNUS.
 * `find -i ABC Pte Ltd` followed by `delete -i 1` deletes the 1st internship in the results of the `find` command.
 
-### Sorting persons by name: `sortp n/`
+### Sorting persons by name: `sort -p n/`
```

Comment:
I was thinking to combine `n/` and `c/` and using them as optional arguments like this:
```suggestion
### Sorting persons by name or by company name: `sort -p [n/] [c/]`
```
Then in the description say `At least one of the optional fields must be provided`, similar to the `edit` commands.

Reply:
This is more complicated to combine since the command only accepts exactly 1 prefix

Reply:
Oh yeah, what if the description says `Exactly one of the optional fields must be provided` instead?

Coz for `sort -i` I thought it'd be a bit messy to have 3 separate descriptions for the different criteria

Reply:
Ya it is quite complicated cause it might be confusing if it is lumped together but also feels pointless to repeat it 3 times